### PR TITLE
Optimaztion: no longer use sed command

### PR DESCRIPTION
**Pull request by GLYASAI**
no longer use sed command

### CODE REVIEW

File: builder/exector/export_app.go [hunk 1]
```diff
@@ -640,7 +640,11 @@ func (i *ExportApp) buildDockerComposeYaml() error {
 		return err
 	}
 
-	err = ioutil.WriteFile(fmt.Sprintf("%s/docker-compose.yaml", i.SourceDir), content, 0644)
+	// sed -i 's/""//g' docker-compose.yaml
+	contentStr := strings.ReplaceAll(string(content), "\"\"", "")
+	// sed -i "s/\*\*None\*\*/$(uuidgen | tr -d -)/g" docker-compose.yaml
+	contentStr = strings.ReplaceAll(contentStr, "**None**", util.NewUUID())
```
Comment (barnettZQG) [inline builder/exector/export_app.go:646]: should only parse env values. 8 bytes?

File: builder/exector/export_app.go [hunk 2]
```diff
@@ -640,7 +640,11 @@ func (i *ExportApp) buildDockerComposeYaml() error {
 		return err
 	}
 
-	err = ioutil.WriteFile(fmt.Sprintf("%s/docker-compose.yaml", i.SourceDir), content, 0644)
+	// sed -i 's/""//g' docker-compose.yaml
+	contentStr := strings.ReplaceAll(string(content), "\"\"", "")
```
Comment (barnettZQG) [inline builder/exector/export_app.go:644]: You should directly prevent volume name= ""

File: builder/exector/export_app.go [hunk 3]
```diff
@@ -571,7 +571,7 @@ func (i *ExportApp) buildDockerComposeYaml() error {
 			volumeName = buildToLinuxFileName(volumeName)
 			volumePath := item.Get("volume_path").String()
 			if item.Get("volume_type").String() == "config-file" {
-				volume := fmt.Sprintf("__GROUP_DIR__/%s/%s:%s", appName, volumePath, volumePath)
+				volume := fmt.Sprintf(".%s:%s", path.Join("./", appName, volumePath), volumePath)
```
Comment (barnettZQG) [inline builder/exector/export_app.go:574]: The appName maybe Chinese, so it is not appropriate to appear directly in the path.

Comment (GLYASAI) [reply builder/exector/export_app.go:574]: appName will be converted to pinyin

File: builder/exector/export_app.go (hunk 2 above)
Comment (GLYASAI) [reply builder/exector/export_app.go:644]: I have no idea to do this after googling, do you know how to do it?

File: builder/exector/export_app.go (hunk 1 above)
Comment (GLYASAI) [reply builder/exector/export_app.go]: fixed

File: builder/exector/export_app.go (hunk 2 above)
Comment (GLYASAI) [reply builder/exector/export_app.go]: fixed

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**